eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/group/ClientSessionGroupMapping.java

```diff
@@ -133,11 +133,11 @@ public void operationComplete(ChannelFuture future) throws Exception {
             return;
         }
 
-        closeSession(session);
-
         //remove session from sessionTable
         sessionTable.remove(addr);
 
+        closeSession(session);
+
```

Comment:
When the client channel is inactive and the session is closed, the client reconnects and the closed session is acquired and used. So first sessionTable.remove, then closeSession.

Reply:
> Can you describe the specific problem, I don't understand what the original way would cause.

When the client channel is inactive and the session is closed, the client reconnects and the closed session is acquired and used. So first sessionTable.remove, then closeSession.

Reply:
When the client reconnects, the port will change, and the session is identified by ip and port. EventMesh acquires session as follows:
```
public Session getSession(ChannelHandlerContext ctx) {
        Session session = getSession((InetSocketAddress) ctx.channel().remoteAddress());
        return session;
    }

    public Session getSession(InetSocketAddress address) {
        return sessionTable.get(address);
    }
```

Reply:
> When the client reconnects, the port will change, and the session is identified by ip and port. EventMesh acquires session as follows:
> 
> ```
> public Session getSession(ChannelHandlerContext ctx) {
>         Session session = getSession((InetSocketAddress) ctx.channel().remoteAddress());
>         return session;
>     }
> 
>     public Session getSession(InetSocketAddress address) {
>         return sessionTable.get(address);
>     }
> ```

Yes. If there is a problem of port reuse, is it better to adjust this way?